crates/atuin-dotfiles/src/shell/nu.rs

```diff
@@ -0,0 +1,8 @@
+pub async fn var_config() -> String {
+    // Because nushell won't autoupdate, we just parse the output of `atuin dotfiles var list` in
+    // nushell and load the env vars that way
+
+    String::from(
+        r#"atuin dotfiles var list | lines | parse "export {name}={value}" | reduce -f {} {|it, acc| $acc | upsert $it.name $it.value} | load-env"#,

```

Comment:
Does nushell have the same shellvar/env var difference?

`atuin dotfiles var list` will output 

```
foo=bar
```

for a shellvar, and 

```
export foo=var
```

for an env var

We'd probably want to handle both cases

Reply:
The only way I've found to do local variables would be to update the plugin file after it runs and waits till the user opens a new shell. That would fix the updating problem and alias problems.

Reply:
I think that makes the most sense, in lieu of being able to load anything dynamically!